### ISSUE TITLE
Installing Kills Admin 5.2.3

### ISSUE THREAD

**Issue by ioyeti777** [AttackStrawbery/iobroker.sma-speedwire#14]
Installing from the given link installs Version 0.6.10 and kills Admin 5.2.3. on other ioB systems I have running Version 0.7.0 without this problem.

index.js:18 iobroker.admin@5.2.3
socket.io.js:59 [2022-01-10T18:25:37.673Z] Try to connect
Adapters.js:447 [ADAPTERS] getAdapters
Adapters.js:654 [ADAPTERS] getAdaptersInfo
Adapters.js:1466 [ADAPTERS] Render
players.png:1 Failed to load resource: the server responded with a status of 404 ()
react-dom.production.min.js:216 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7BAdapter%20to%20query%20values%20from%20sma%20devices%7D for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Ec (5.6300204f.chunk.js:2)
    at 5.6300204f.chunk.js:2
    at Po (5.6300204f.chunk.js:2)
    at Zi (5.6300204f.chunk.js:2)
    at jl (5.6300204f.chunk.js:2)
    at Sl (5.6300204f.chunk.js:2)
    at Cl (5.6300204f.chunk.js:2)
    at zl (5.6300204f.chunk.js:2)
    at 5.6300204f.chunk.js:2
    at t.unstable_runWithPriority (5.6300204f.chunk.js:2)

**Comment by Apollon77** [comment]
Yes reason is an error in io-package ... there is one level too much: https://github.com/AttackStrawbery/ioBroker.sma-speedwire/blob/release/io-package.json#L16

**Comment by rg-engineering** [comment]
solved under https://github.com/rg-engineering/ioBroker.sma-speedwire ... but still a lot of to do's...
